react-hyf/react-hyf-client/src/ProfileGenerator.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+
+class ProfileGenerator extends Component {
+  constructor() {
+    super();
+    this.state = {
+      error: null,
+      isLoaded: false,
+      people: [],
+      randomPerson: '',
```

Comment:
`randomPerson` is an object right? Better to set it's value to `null` here, not a string, this makes it seem as if `randomPerson` is a string the whole time.